src/server/routes/parcel.routes.spec.ts

```diff
@@ -152,4 +161,85 @@ describe('parcel route', () => {
     expect(response).toHaveProperty('statusCode', HTTP_STATUS_CODES.ACCEPTED);
     expect(logs).toContainEqual(partialPinoLog('info', 'Ignoring invalid service message'));
   });
+
+  describe('store PDA', () => {
+    test('Valid PDA should be stored', async () => {
+      const certificatePath = await generatePDACertificationPath(KEY_PAIR_SET);
+      const pdaPath = new CertificationPath(certificatePath.pdaGrantee, [
+        certificatePath.privateEndpoint,
+        certificatePath.privateGateway,
+      ]);
+      const privateEndpoint = new PrivateEndpointConnParams(
+        PRIVATE_ENDPOINT_KEY_PAIR.privateGateway.publicKey,
+        PRIVATE_ENDPOINT_ADDRESS,
+        pdaPath,
+      );
+
+      const { parcelSerialized } = await generateParcel(
+        parcelRecipient,
+        KEY_PAIR_SET,
+        new Date(),
+        {
+          publicKey,
+          keyId: sessionKey.keyId,
+        },
+        'application/vnd+relaycorp.awala.pda-path',
+        Buffer.from(await privateEndpoint.serialize()),
+      );
+      const spyOnSavePrivateEndpointChannel = jest.spyOn(
+        activeEndpoint,
+        'savePrivateEndpointChannel',
+      );
+
+      const response = await server.inject({
+        ...validRequestOptions,
+        payload: parcelSerialized,
+      });
+
+      expect(spyOnSavePrivateEndpointChannel).toHaveBeenCalledOnce();
+      expect(response).toHaveProperty('statusCode', HTTP_STATUS_CODES.ACCEPTED);
+      expect(logs).toContainEqual(
+        partialPinoLog('info', 'Private endpoint connection params stored'),
+      );
+    });
+
+    test('Invalid PDA should be accepted but not stored', async () => {
```

Comment:
The connection params contains a PDA, and what we're interested in is if the connection params (including the PDA) is valid.

```suggestion
    test('Invalid connection params should be accepted but not stored', async () => {
```